cli/deploy-custom-container-minimal-multimodel.sh

```diff
@@ -70,10 +73,12 @@ SCORING_URL=$(az ml online-endpoint show -n $ENDPOINT_NAME --query scoring_uri -
 echo "Scoring url is $SCORING_URL"
 
 # <test_online_endpoints> 
-curl -H "Authorization: Bearer $KEY" -H "Content-Type: application/json" -d @"$BASE_PATH/test-data/test-data-iris.json"  $SCORING_URL
-curl -H "Authorization: Bearer $KEY" -H "Content-Type: application/json" -d @"$BASE_PATH/test-data/test-data-diabetes.json"  $SCORING_URL
+curl -H "Authorization: Bearer $KEY" -H "Content-Type: application/json" -d @"$BASE_PATH/test-data/iris-test-data.json"  $SCORING_URL
+curl -H "Authorization: Bearer $KEY" -H "Content-Type: application/json" -d @"$BASE_PATH/test-data/diabetes-test-data.json"  $SCORING_URL
 # </test_online_endpoints> 
 
 # <delete_online_endpoint>
 az ml online-endpoint delete -y -n $ENDPOINT_NAME
 # </delete_online_endpoint>
+
+rm *.yml_
```

Comment:
Do we need `rm *.yml_`?
```suggestion

```

Reply:
Changes have been made... runners still having problems it looks like.